supervisor/core.py

```diff
@@ -116,14 +116,14 @@ async def setup(self):
             self.sys_host.load(),
             # Adjust timezone / time settings
             self._adjust_system_datetime(),
+            # Load CPU/Arch
+            self.sys_arch.load(),
```

Comment:
Kind of a scary change moving things around in core setup but I think it makes sense. Looking at `CpuArch` its only dependencies in `coresys` appear to be `sys_supervisor` and `sys_machine`, both of which are set up before this method runs.

Alternatively I can adjust all the plugins to pass in `self.sys_supervisor.arch` when they call `install` but using `self.sys_arch.default` seemed more correct.